CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to PowerShell Koans (PSKoans)
+
+Welcome to PSKoans and thank you for your interest in contributing!
+
+There are many ways that you can contribute, beyond writing or coding. The goal of this document is to provide a high-level overview of how you can get involved and how to interact.
+
+## PSKoans Development Tools
+
+If you want to contribute, at the very least, you'll need: 
+- [ ] [A GitHub Account](http://github.com)
+
+Here are a list of programs that should make contributing easier:
+- [ ] [GitHub Desktop](https://desktop.github.com)
+- [ ] [Visual Studio Code](https://code.visualstudio.com)
+
+Some useful VSCode extensions are as follows:
+- [ ] [PowerShell Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell)
+- [ ] [Code Spell Checker Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
+
+## PSKoans Development and Contribution Steps
+
+1. Fork a copy of the [vexx32/PSKoans](https://github.com/vexx32/PSKoans) repository using your GitHub account
+2. Clone your PSKoans fork locally to your computer
+    > Tip: You may want to clone it to your OneDrive folder so that it's available on all your shared systems.
+    1. This can be done either through Git directly or with GitHub Desktop.
```

Comment:
So I actually have zero experience with git command line. Writing this Contributing guide sounds like a good time to learn!

Reply:
Ok, so I just did a writeup for the git lifecycle with GitHub Desktop, git with PowerShell, and VSCode. 

I'm sure there are mistakes in there and ways to clean it up and make it look nice, but its in there, and basically tested because I had to do it for myself to be able to write about it hahaha.